codacy/templates/fluentd/fluentd-conf.yaml

```diff
@@ -14,28 +14,28 @@ data:
     </match>
 
     # Remove unneeded fields from the logs
-    <filter **>
+    <filter kubernetes.var.log.containers.**{{ .Release.Namespace }}**>
```

Comment:
This makes our fluentd fetch logs files from the codacy installation namespace only

Reply:
Sadly we can't do this yet. We suspect it is due to the fact that our version of fluentd doesn't support this sort of syntax. This will be taken care of in another PR